react-components/src/architecture/concrete/observationsDomainObject/ObservationsTool.ts

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type IconType } from '@cognite/cogs.js';
+import { type TranslateKey } from '../../base/utilities/TranslateKey';
+import { type Observation } from './models';
+import { ObservationsDomainObject } from './ObservationsDomainObject';
+import { type Overlay3D } from '@cognite/reveal';
+import { DEFAULT_OVERLAY_COLOR, SELECTED_OVERLAY_COLOR } from './constants';
+import { BaseEditTool } from '../../base/commands/BaseEditTool';
+import { type VisualDomainObject } from '../../base/domainObjects/VisualDomainObject';
+
+export class ObservationsTool extends BaseEditTool {
+  private _selectedOverlay: Overlay3D<Observation> | undefined;
```

Comment:
_selectedOverlay. This should be in the view. Remember that there should be no visualization code in the tools.